doc/reference/configuration/cfg_replication.rst

```diff
@@ -708,3 +712,54 @@
     | Default: 'soft'
     | Environment variable: TT_ELECTION_FENCING_MODE
     | Dynamic: yes
+
+.. _cfg_replication-instance_name:
+
+.. confval:: instance_name
+
+    Since version :doc:`3.0.0 </release/3.0.0>`.
+
+    Specify the instance name.
+    This value must be unique in a replica set.
```

Comment:
Should we highlight that the name is persisted (written into the database)?

Should we mention which names are valid and which are invalid (see #3468)?

Should we clarify when the name can be set or deleted (see #3468)?

(The same question about the replicaset and the cluster names.)

Reply:
All done, thanks!

> Should we mention which names are valid and which are invalid (see https://github.com/tarantool/doc/issues/3468)?

This raises the question: are the rules applied to group/replica set/instance names for a new configuration approach are the same? If yes, I should update the reference: 

- https://www.tarantool.io/en/doc/latest/reference/configuration/configuration_reference/#configuration-reference-groups-name
- https://www.tarantool.io/en/doc/latest/reference/configuration/configuration_reference/#configuration-reference-replicasets-name
- https://www.tarantool.io/en/doc/latest/reference/configuration/configuration_reference/#configuration-reference-instances-name